### PR TITLE
build: generate manpages

### PR DESCRIPTION
**Pull request by teg**
This was done manually in the specfile, let meson do it.

This addresses issue #48.

Signed-off-by: Tom Gundersen <teg@jklm.no>

### CODE REVIEW

File: docs/meson.build [hunk 1]
```diff
@@ -0,0 +1,19 @@
+custom_target(
+        'man_dbus_broker',
+        output : 'dbus-broker.1',
+        input : 'dbus-broker.rst',
+        command : [rst2man, '@INPUT@', '@OUTPUT@'],
+        install : true,
+        install_dir : join_paths(get_option('mandir'), 'man1')
```
Comment (dvdhrm) [inline docs/meson.build]: We don't put spaces before the colon ;)

Also, doesn't this need: `build_by_default: true`
Or is the `install: true` enough to pull this in?

Comment (teg) [reply docs/meson.build]: It appears to be enough.

Comment (teg) [reply docs/meson.build]: Fixed the whitespace.

**Comment by teg** [comment]
Added python-docutils to build requirements (which provides rst2man).

**Pull request merged**